frontend/src/components/Pages/InspectionPage/InspectionSection.tsx

```diff
@@ -64,12 +64,25 @@ export interface OngoingMissionType {
     [missionId: string]: boolean
 }
 
-export const compareInspections = (i1: Inspection, i2: Inspection) => {
-    if (!i1.missionDefinition.inspectionFrequency) return 1
-    if (!i2.missionDefinition.inspectionFrequency) return -1
-    if (!i1.missionDefinition.lastRun) return 1
-    if (!i2.missionDefinition.lastRun) return -1
-    else return i1.deadline!.getTime() - i2.deadline!.getTime()
+export const compareInspections = (inspection1: Inspection, inspection2: Inspection) => {
```

Comment:
Can the name of this function be improved? I'm not able to read this name and understand what the output implies. Diving into the logic is complex so maybe a more descriptive name could help.

Reply:
I've divided it up into a new function now

Reply:
I've added a function comment, I hope that helps?